node/parallel/src/chain_spec/heiko.rs

```diff
@@ -161,10 +164,25 @@ fn testnet_genesis(
                     .collect()
             },
         },
-        // TODO collateral selection
-        pallet_aura: AuraConfig {
-            authorities: initial_authorities.iter().map(|x| (x.1.clone())).collect(),
+        pallet_collator_selection: CollatorSelectionConfig {
+            invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+            candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
```

Comment:
There is a potential bug in this pallet, `candidacy_bond` is not reserved in genesis config.

Reply:
@kaichaosun can you explain more on this? normally `candidacy_bond` will only be used while registering a new candidate

Reply:
Are there more context about this one?